data/langdb.yaml

```diff
@@ -493,6 +493,7 @@ languages:
   tl: [Latn, [AS], Tagalog]
  # A very complicated case. Names.php is Cyrl. In TWN they argue about Cyrl, Latn, and Arab. I can't find reliable external sources. --Amir
   tly: [Cyrl, [EU, AS, ME], толышә зывон]
+  tmr: [Hebr, [ME, EU, AM], Jewish Babylonian Aramaic]
```

Comment:
TLDR:
1. AM can be there.
2. It needs an autonym and not an English name. The autonym should probably be ארמית בבלית.

Now, the longer version:

I'll think out loud for a minute. This is an ancient language, which is not used today for new writing. It appears in important religious books, which are studied in all Jewish religious schools all over the world, including Israel, U.S., France, and many other countries. In this sense, this language is indeed used on these continents, although I'm wondering what will it be used for in ULS. It won't be useful for writing a new Wikipedia or for translating MediaWiki software messages. It can be useful for texts in Wikisource, but such texts are stored in the Hebrew Wikisource anyway. It uses the same alphabet as Hebrew, so it doesn't need its own IME entry. So it will never appear in a ULS panel for any of these things. The only thing in Wikimedia context I can think of where this will be really useful is Wikidata labels, and these can actually be relevant for people who edit from America. So OK, it can say AM here :)

It also needs a proper autonym and not an English name, probably ארמית בבלית. I'm not sure that this is really the _correct_ name of the language in itself, but for all practical purposes, people who deal with this language will know it by this Hebrew name (or an even shorter ארמית, but this is only "Aramaic", and there are other kinds of Aramaic).